api/build.go

```diff
@@ -362,6 +362,106 @@ func GetBuilds(c *gin.Context) {
 	c.JSON(http.StatusOK, b)
 }
 
+// swagger:operation GET /api/v1/repos/{org} builds GetOrgBuilds
+//
+// Create a build in the configured backend
```

Comment:
This comment is incorrect.
```suggestion
// Get a list of builds by org in the configured backend
```